host/src/components/FeaturedMistakes.jsx

```diff
@@ -0,0 +1,163 @@
+import React, { useEffect, useState } from "react";
```

Comment:
Featured Mistakes component that allows for teachers to preview the incoming student answers and select which ones they want to display in phase 2 as options for students to answer. See the PR description for a screenshot and a Figma link